ckan/model/activity.py

```diff
@@ -180,14 +211,12 @@ def _group_activity_query(group_id):
         # Return a query with no results.
         return model.Session.query(model.Activity).filter("0=1")
 
-    dataset_ids = [dataset.id for dataset in group.packages()]
+    q = model.Session.query(
+        model.Activity
+    ).filter(
+        model.Activity.object_id == group_id
+    )
```

Comment:
Nope, because the behaviour of the two is not the same. Just this change would leave you with no dataset activity.